### PR TITLE
Command UI improvement proposal

### PR DESCRIPTION
**Pull request by AlexMartinFR**
## Changes:

- Official dependencies names displayed instead of their class names (No more visual confusion between MsSql & MySql)
- Line break to separate the exit menu item from the rest
- Padding around the menu to help the text breathe

## Improvements:

I think the UI would be awesome with greater vertical padding around the menu items text.
I haven't found a way to do so.
Any suggestion?

## Preview:
image

### CODE REVIEW

File: app/Commands/EnableCommand.php [hunk 1]
```diff
@@ -37,7 +40,8 @@ public function handle(): void
     public function enableableServices(): array
     {
         return collect((new Services)->all())->mapWithKeys(function ($fqcn, $shortName) {
-            return [$shortName => Str::afterLast($fqcn, '\\')];
+            $fullName = Str::of($fqcn)->replace(['MsS', 'Sql'], ['MS S', 'SQL']);
```
Comment (mattstauffer) [inline app/Commands/EnableCommand.php:43]: Would you pull this out to a private method?

Thanks!

Comment (AlexMartinFR) [reply app/Commands/EnableCommand.php:43]: Sure!

**Comment by AlexMartinFR** [comment]
WIP

I'm toying with something better to also sort services by category.

**Comment by mattstauffer** [comment]
Thanks @AlexMartinFR!

**Pull request merged**